sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/rest_client.hpp

```diff
@@ -452,7 +452,7 @@ namespace Azure { namespace Storage { namespace Blobs {
        */
       Nullable<DateTime> DeletedOn;
       /**
-       * Remaining days before this container will be permanantely deleted. Only valid when this
+       * Remaining days before this container will be permanently deleted. Only valid when this
```

Comment:
I believe this is code-gen'd and will be overwritten. This needs to be fixed on the swagger/transform side in markdown.

For now, consider excluding it from the PR.

cc @Jinming-Hu

Reply:
I can't find any occurrences of "permanantely" in the Swaggers: https://github.com/search?q=repo%3AAzure%2Fazure-rest-api-specs%20permanantely&type=code

Reply:
This has been fixed in swagger in PR https://github.com/Azure/azure-sdk-for-cpp/pull/5208.

I just didn't re-generate rest-client files.